proxy.conf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "/api/v1": {
+    "logLevel": "debug",
+    "changeOrigin": true,
+    "target": "http://localhost:3000"
```

Comment:
NOTE: To avoid CORS issue with cookies, you can use the new `npm run start:proxy` script.